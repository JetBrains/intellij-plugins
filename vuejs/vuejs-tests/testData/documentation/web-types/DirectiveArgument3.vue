<template>
  <div v-cool:foo<caret>=""
</template>
