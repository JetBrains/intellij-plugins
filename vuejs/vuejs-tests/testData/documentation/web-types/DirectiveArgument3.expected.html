<div class='definition'><pre>Vue directive argument</pre></div>
<div class='content'>
<p>Cool argument</p>
</div><table class='sections'>
<tr><td valign='top' class='section'><p>Pattern:</td><td valign='top'>\[a-z]*\</td>
<tr><td valign='top' class='section'><p>Library:</td><td valign='top'><p>test-lib@2.0.0-rc.24</td>
</table>